src/pages/getting-started/app-guides/agent-app-widgets/index.mdx

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Did you use `git mv` for this content. It looks like it was created, but should be just moved. Same for other files.